src/omv/modules/py_ml.c

```diff
@@ -138,13 +174,24 @@ static const mp_obj_type_t py_ml_model_type;
 
 static void py_ml_model_print(const mp_print_t *print, mp_obj_t self_in, mp_print_kind_t kind) {
     py_ml_model_obj_t *self = MP_OBJ_TO_PTR(self_in);
-    mp_printf(print,
-              "{size: \"%d\", ram: \"%d\","
-              " inputs_size: \"%d\", input_dtype: \"%c\", input_scale: \"%f\", input_zero_point: \"%d\","
-              " outputs_size: \"%d\" output_dtype: \"%c\", output_scale: \"%f\", output_zero_point: \"%d\"}",
-              self->size, self->memory_size, self->inputs_size, self->input_dtype,
-              (double) self->input_scale, self->input_zero_point, self->outputs_size, self->output_dtype,
-              (double) self->output_scale, self->output_zero_point);
+    mp_printf(print, "{size: %d, ram: %d", self->size, self->memory_size);
```

Comment:
You removed the quotes that you specifically asked me to put back ?

Reply:
Added back.

Reply:
Then let's remove them, I wasn't sure why we needed them to begin with.

Reply:
Done